src/api/nodeRpcCall.js

```diff
@@ -233,59 +233,11 @@ const getAdditionalAssets = async (address, isIndexNeed = false) => {
   }
 };
 
-const bridgeSubscribe = async (asset, receipt_id, onChange) => {
-  const api = await getApi();
-  let bridge;
-  /* eslint-disable eqeqeq */
-  if (asset == 'LLM') bridge = api.query.ethLLMBridge;
-  else if (asset == 'LLD') bridge = api.query.ethLLDBridge;
-  // returns unsub func
-  return {
-    unsubscribe: await bridge.statusOf(receipt_id, onChange),
-  };
-};
-
-const bridgeDeposit = async ({ asset, amount, ethereumRecipient }, walletAddress) => {
-  const api = await getApi();
-  let bridge;
-  if (asset == 'LLM') bridge = api.tx.ethLLMBridge;
-  else if (asset == 'LLD') bridge = api.tx.ethLLDBridge;
-  // eslint-disable-next-line no-undef
-  else throw new Exception('Unknown asset');
-
-  const call = bridge.deposit(amount, ethereumRecipient);
-  return submitExtrinsic(call, walletAddress, api);
-};
-
-const bridgeWithdraw = async ({ receipt_id, asset }, walletAddress) => {
-  const api = await getApi();
-  let bridge;
-  if (asset == 'LLM') bridge = api.tx.ethLLMBridge;
-  else if (asset == 'LLD') bridge = api.tx.ethLLDBridge;
-  // eslint-disable-next-line no-undef
-  else throw new Exception('Unknown asset');
-
-  const call = bridge.withdraw(receipt_id);
-  return submitExtrinsic(call, walletAddress, api);
-};
-
-const bridgeConstants = async (asset) => {
-  const api = await getApi();
-  let bridge;
-  if (asset == 'LLM') bridge = api.consts.ethLLMBridge;
-  else if (asset == 'LLD') bridge = api.consts.ethLLDBridge;
-  /* eslint-enable eqeqeq */
-  // eslint-disable-next-line no-undef
-  else throw new Exception('Unknown asset');
-
-  return bridge;
-};
-
 const provideJudgementAndAssets = async ({
   address, hash, walletAddress, merits, dollars,
 }) => {
-  merits = parseMerits(merits)
-  dollars = parseDollars(dollars)
+  const parsedMerits = parseMerits(merits);
+  const parsedDollars = parseDollars(dollars);
```

Comment:
Is this on purpose or some merge issue ?

Reply:
Yep does are eslint fixes

Reply:
No they are not, linter will not rename merits to parsedMerits